lib/util.js

```diff
@@ -124,7 +124,7 @@ function generatePreviewHead(parsedConfig) {
 
   for(const key of Object.keys(parsedConfig)) {
     for(const value of parsedConfig[key]) {
-      if(key == 'script') {
+      if(key == 'script' && value.scr) {
```

Comment:
Is this a typo or are you sure about `scr` or maybe `src`?

Reply:
Typo. Fixed. While this may be a minor change, I'm not sure it has fixed my issues. Hold off, for now, on it.